ocrd_olahd_client/processor.py

```diff
@@ -21,12 +22,18 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
     def process(self):
+        LOG = getLogger('processor.OlaHdClientProcessor')
+        LOG.info('Posting workspace to %s' % self.parameter['endpoint'])
         client = OlaHdClient(self.parameter['endpoint'], self.parameter['username'], self.parameter['password'])
         bagger = WorkspaceBagger(Resolver(), strict=True)
         # TODO
         dest = join(gettempdir(), 'bag-%d.ocrd.zip' % int(round((time() * 1000))))
         # TODO
         ocrd_identifier = self.workspace.mets.unique_identifier
+        LOG.info('Bagging workspace')
         bagger.bag(self.workspace, ocrd_identifier, dest=dest)
+        LOG.info('Logging in')
         client.login()
+        LOG.info('POST bag')
```

Comment:
```suggestion
        LOG.debug('Bagging workspace')
        bagger.bag(self.workspace, ocrd_identifier, dest=dest)
        LOG.debug('Logging in')
        client.login()
        LOG.debug('POST bag')
```

Reply:
Personally I prefer not to log too much to not make the code harder to read. I don't read logs often (except error logs) and prefer to debug in those cases. I am ok with it tough but would rather remove it or set parts of it to debug.

Reply:
Less noise is better, agreed.